src/plugins/data/public/search/session/sessions_mgmt/components/table/table.test.tsx

```diff
@@ -175,13 +175,15 @@ describe('Background Search Session Management Table', () => {
             />
           </LocaleWrapper>
         );
-        jest.advanceTimersByTime(20000);
-      });
-
-      // 1 for initial load + 2 refresh calls
-      expect(sessionsClient.find).toBeCalledTimes(3);
 
-      jest.useRealTimers();
```

Comment:
Is this still needed?

Reply:
it works without, but you're right, once we use a fake timer in a file, we should take care of switching to the "real" one to omit troubles